mkt/search/serializers.py

```diff
@@ -65,34 +65,13 @@ def get_fields(self):
         for key, field in fields.items():
             if isinstance(field, TranslationSerializerField):
                 fields[key] = ESTranslationSerializerField(source=field.source)
-                fields[key].initialize(parent=self, field_name=key)
         return fields
 
-    @property
-    def data(self):
-        """
-        Returns the serialized data on the serializer.
-        """
-        if self._data is None:
-            if self.many:
-                self._data = [self.to_native(item) for item in self.object]
-            else:
-                self._data = self.to_native(self.object)
-        return self._data
-
-    def field_to_native(self, obj, field_name):
-        # DRF's field_to_native calls .all(), which we want to avoid, so we
-        # provide a simplified version that doesn't and just iterates on the
-        # object list.
-        if hasattr(obj, 'object_list'):
-            return [self.to_native(item) for item in obj.object_list]
-        return super(BaseESSerializer, self).field_to_native(obj, field_name)
```

Comment:
yes, `.all()` is only called on django managers now: https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/serializers.py#L611